src/main/java/com/trainer/name/service/TrainerService.java

```diff
@@ -113,4 +113,11 @@ private void validateUpdateParameters(String name, String email, Trainer existin
         }
     }
 
+    public void delete(int id) throws TrainerNotFoundException {
```

Comment:
TrainerNotFoundExceptionは実行時例外を継承しているのでthrowsはなくてもいいですね。
```suggestion
    public void delete(int id) {
```

Reply:
throwsを削除し、その後もきちんと動作することを確認しました
4753b38c999ddac8ccbf3e0bf74e066df31299a1
![image](https://github.com/Ito-0131/10th-Assignment/assets/138368847/9a16e6ac-a382-4bcd-a910-bc042eebcae5)
![image](https://github.com/Ito-0131/10th-Assignment/assets/138368847/1f227afe-a834-4e5e-ba90-a4b7df475611)

Reply:
動作確認ぬからず行っていて素晴らしい！